cvmfs/cvmfs_config

```diff
@@ -1564,6 +1587,16 @@ case "$1" in
      cvmfs_bugreport $@
      RETVAL=$?
      ;;
+     
+  fsck)
+    if [ `id -u` -ne 0 ]; then
+      echo "root privileges required"
+      exit 1
+    fi
+    shift 1
+    cvmfs_fsck
```

Comment:
It just comes to my mind that it would be nice to pass options to `cvmfs_fsck`.  Like `cvmfs_fsck "$@"` and inside the `cvmfs_fsck` this could be passed to /usr/bin/cvmfs_fsck.